pkg/cmd/streamsDelete_test.go

```diff
@@ -60,3 +60,21 @@ func TestStreamsDeleteNamespaceNameFlagsReturnsOk(t *testing.T) {
 
 	assert.Nil(t, err)
 }
+
+func TestStreamsDeleteNamespaceWithoutNameFlagReturnsError(t *testing.T) {
```

Comment:
No need to mention namespace in this function name since we're not working with namespaces anywhere